Library/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -24,39 +30,61 @@ public protocol SettingsAccountViewModelType {
 public final class SettingsAccountViewModel: SettingsAccountViewModelInputs,
   SettingsAccountViewModelOutputs, SettingsAccountViewModelType {
   public init(_ viewControllerFactory: @escaping (SettingsAccountCellType, Currency) -> UIViewController?) {
-    let userAccountFields = self.viewWillAppearProperty.signal
-      .switchMap { _ in
-        AppEnvironment.current.apiService
-          .fetchGraphUserAccountFields(query: UserQueries.account.query)
-          .materialize()
-      }
+    let userAccountFields = Signal.merge(
+      self.viewDidLoadProperty.signal,
+      self.viewWillAppearProperty.signal.skip(first: 1)
+    )
+    .switchMap { _ in
+      AppEnvironment.current.apiService
+        .fetchGraphUserAccountFields(query: UserQueries.account.query)
+        .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+        .materialize()
+    }
 
     self.fetchAccountFieldsError = userAccountFields.errors().ignoreValues()
 
-    let shouldHideEmailWarning = userAccountFields.values()
-      .map { response -> Bool in
-        guard let isEmailVerified = response.me.isEmailVerified,
-          let isDeliverable = response.me.isDeliverable else {
+    let user = userAccountFields.values().map(\.me)
+
+    let shouldHideEmailWarning = user
+      .map { user -> Bool in
+        if user.isAppleConnected == .some(true) {
+          return true
+        }
+
+        guard let isEmailVerified = user.isEmailVerified,
```

Comment:
For your consideration:
```swift
if [user.isEmailVerified, user.isDeliverable]
  .map({ $0 ?? false })
  .allSatisfy(isTrue) {
  return true
}
```
Not sure how much it really adds for readability, just let `guard let`s.

Reply:
Hmm yeah I'm not sure the extra mental overhead is worth it here 😅